packages/vouching/contracts/Vouching.sol

```diff
@@ -0,0 +1,101 @@
+pragma solidity ^0.4.24;
+
+
+import "./ZepToken.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+contract Vouching {
+  using SafeMath for uint256;
+
+  struct Dependency {
+    address owner;
+    address dependencyAddress;
+    uint256 stake;
+  }
+
+  mapping (string => Dependency) private _registry;
+  uint256 private _minimumStake;
+  ZepToken private _token;
```

Comment:
Let's use a generic ERC20 here

Reply:
In particular, we might need to call other specific ZEP methods when doing the TPL integration..

Reply:
As discussed offline, we'll move to use a generic ERC20 here to keep things decoupled.